website/source/docs/templates/post-processors.html.markdown

```diff
@@ -96,6 +98,27 @@ compressed then uploaded, but the compressed result is not kept.
 As you may be able to imagine, the **simple** and **detailed** definitions are
 simply shortcuts for a **sequence** definition of only one element.
 
+## Creating Vagrant Boxes in Atlas
+
+It is important to sequence post processors when creating and uploading vagrant boxes to Atlas via Packer. Using a sequence will ensure that the post processors are ran in order and creates the vagrant box prior to uploading the box to Atlas.
+
+``` {.javascript}
+{
+  "post-processors": [
+    [
+      {
+        "type": "vagrant",
+        "keep_input_artifact": false
+      },
+      {
+        "type": "atlas",
+        ...
```

Comment:
We could also link to this page: https://packer.io/docs/post-processors/atlas.html